RecoEgamma/EgammaElectronProducers/plugins/ElectronSeedProducer.cc

```diff
@@ -297,29 +257,11 @@ void ElectronSeedProducer::filterClusters(const reco::BeamSpot& bs,
         hoe2s.push_back(std::numeric_limits<float>::infinity());
       }
     }
-
-    if (applySigmaIEtaIEtaCut_ == true) {
-      noZS::EcalClusterLazyTools lazyTool_noZS(event, setup, ebRecHitCollection_, eeRecHitCollection_);
-      std::vector<float> vCov = lazyTool_noZS.localCovariances(*(scl.seed()));
-      int detector = scl.seed()->hitsAndFractions()[0].first.subdetId();
-      if (detector == EcalBarrel)
-        sigmaIEtaIEtaEB_.push_back(edm::isNotFinite(vCov[0]) ? 0. : sqrt(vCov[0]));
-      if (detector == EcalEndcap)
-        sigmaIEtaIEtaEE_.push_back(edm::isNotFinite(vCov[0]) ? 0. : sqrt(vCov[0]));
-    }
   }
   LogDebug("ElectronSeedProducer") << "Filtered out " << sclRefs.size() << " superclusters from "
                                    << superClusters->size();
-}
 
-void ElectronSeedProducer::filterSeeds(edm::Event& event,
```

Comment:
The `filterSeeds` member function got integrated into `produce`, so `theInitialSeedColl` and `theInitialSeedColls` don't have to be members but can be local variables without bloating this function with more arguments.